server/service/src/vaccination/update/generate.rs

```diff
@@ -94,6 +94,7 @@ pub fn generate(
                     reason_id: None,
                     note: None,
                 }],
+                status: None,
```

Comment:
This None status will default to new, and doesn't change existing insert behaviour from vaccines.

Reply:
We would want to finalise this as well.